### PR TITLE
Development cleanup

### PR DESCRIPTION
**Pull request by Phhere**
Some update for the build and development tools
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/derchrisuk%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/226%23issuecomment-94917450%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-04-21T19%3A48%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%200239b7e92752afbbb2086c6760ae080a52f318f0%20nightwatch.json%2011%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/226%23discussion_r28308541%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Shouldn%27t%20that%20be%20singular%20-%20%60/development/%60%3F%22%2C%20%22created_at%22%3A%20%222015-04-14T08%3A28%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22yes%22%2C%20%22created_at%22%3A%20%222015-04-14T08%3A32%3A06Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-04-21T19%3A47%3A53Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20nightwatch.json%3AL10-22%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/webSPELL/webSPELL/pull/226%23issuecomment-94917450%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/226%23discussion_r28308541%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/226%23discussion_r28308743%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/226%23discussion_r28815028%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/derchrisuk'></a>

- [x] <a href='#crh-comment-Pull 0239b7e92752afbbb2086c6760ae080a52f318f0 nightwatch.json 11'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/pull/226#discussion_r28308541'>File: nightwatch.json:L10-22</a></b>
- <a href='https://github.com/Pascalmh'></a> Shouldn't that be singular - `/development/`?
- <a href='https://github.com/Phhere'></a> yes

<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/226?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/226?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/226?approve=0'></a>&nbsp;<a href='https://github.com/webSPELL/webSPELL/pull/226'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by webSPELL-org** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/webSPELL/webSPELL/pull/226'>Refresh</a>.<a href='#crh-update-none'></a>

File: nightwatch.json [hunk 1]
```diff
@@ -10,12 +10,12 @@
...
     "selenium" : {
         "start_process" : true,
-        "server_path" : "./tools/selenium-server-standalone-2.44.0.jar",
+        "server_path" : "./developments/tools/selenium-server-standalone-2.44.0.jar",
         "log_path" : "",
         "host" : "127.0.0.1",
         "port" : 4444,
         "cli_args" : {
-            "webdriver.chrome.driver" : "./tools/chromedriver_linux64",
+            "webdriver.chrome.driver" : "./developments/tools/chromedriver_linux64",
```
Comment (Pascalmh) [inline nightwatch.json]: Shouldn't that be singular - `/development/`?

Comment (Phhere) [inline nightwatch.json]: yes

Comment (derchrisuk) [inline nightwatch.json]: :+1:

**Comment by derchrisuk** [comment]
:shipit:

**Pull request merged**